### ISSUE TITLE
Correct imagenet top-1 accuracy uploaded pretrained weights of hybrid ViT?

### ISSUE THREAD

**Issue by coeusguo** [google-research/vision_transformer#39]
Hi, Thanks so much for the great work!
I tried to restore 'imagenet21k+imagenet2012_R50+ViT-B_16.npz' and got 83.41% imagenet Top-1 accuracy, is that a correct accuracy? I directly resize the test images into 384x384 without a crop, I'm not quite sure whether it is a correct operation.

**Comment by andsteing** [comment]
Top-1 accuracy on `imagenet2012` should be 85.082% for that model.

We use `im = tf.image.resize(im, [384, 384])` and then normalize the image `m = (im - 127.5) / 127.5` - see https://github.com/google-research/vision_transformer/blob/master/vit_jax/input_pipeline.py

**Comment by dribnet** [comment]
My current B_16 top-1 accuracy is also a bit lower at 83.6%. One difference I found looking at this more closely is that the image normalization I was using was based on the `vit_jax.ipynb` sample notebook, which is subtly different:

<img width="778" alt="Screen Shot 2020-12-08 at 7 07 52 AM" src="https://user-images.githubusercontent.com/945979/101388218-83eec000-3924-11eb-8bd4-29110307f820.png">

I've replaced this in my own code now with the `m = (im - 127.5) / 127.5` version and am seeing slight differences and will report back if this closes the gap on the reported score.

**Comment by dribnet** [comment]
So that normalization difference cited above changes very little (0.01%) and so I looked closer at the resizing code, but matching that exactly also only moved my score about 0.4% - the best top-1 accuracy I could get matching both for the ViT-B_16 model was 84.01%. So I'd been keen to do a deeper dive to diagnose this if anyone else has gotten to 85% and would be willing to compare exact results. I'll suggest a good starting point would be to look at the accuracy of a particular class (50 images each) and then potentially dive down into any differences for particular files, etc.
